pennylane/math/__init__.py

```diff
@@ -23,49 +23,45 @@
     Furthermore, the names and behaviour of these functions may differ from similar
     functions in common frameworks; please refer to the function docstrings for more details.
 
-Internally, these functions dispatch by using the :class:`TensorBox` class, a container and API for
-array-like objects that allows array manipulation to be performed in a unified manner for supported
-tensor/array manipulation frameworks.
-
 The following frameworks are currently supported:
 
 * NumPy
 * Autograd
 * TensorFlow
 * PyTorch
+* JAX
 """
+import autoray as ar
+
 from .fn import (
-    T,
-    abs_ as abs,
+    _multi_dispatch,
     allclose,
     allequal,
-    angle,
-    arcsin,
     block_diag,
     cast,
     cast_like,
     concatenate,
-    conj,
     convert_like,
     cov_matrix,
     diag,
     dot,
-    expand_dims,
-    flatten,
-    gather,
     get_interface,
     marginal_prob,
     ones_like,
-    reshape,
     requires_grad,
-    scatter_element_add,
-    shape,
-    sqrt,
     stack,
-    squeeze,
-    sum_ as sum,
-    take,
-    toarray,
     where,
 )
-from .tensorbox import TensorBox, wrap_output
+
+
+def __getattr__(name):
+    if name in globals():
+        return globals()[name]
+
+    if name == "T":
+        return getattr(ar.numpy, "transpose")
+
+    if name == "toarray":
+        return getattr(ar, "to_numpy")
```

Comment:
For backwards compatibility. These two functions have different names in autoray compared to what we used previously.